modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -1170,6 +1170,17 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		_accountGroupRelService.addAccountGroupRel(accountGroupId, className, classPK);
 	}
 
+	public void method(
+		String searchEngineId, long companyId, String className,
+		boolean commitImmediately) {
+
+	IndexWriterHelperUtil.deleteEntityDocuments(companyId, className, commitImmediately);
+	IndexWriterHelperUtil.deleteEntityDocuments("test123", 123, null, null);
+
```

Comment:
Hi @regiCesar21, since is a test for the same JSONObject and the same case, we don't need to separate by this empty line.
And also the message case should also be the first and only one is necessary.